x/did/types/did.go

```diff
@@ -280,7 +280,7 @@ func (doc DIDDocument) validVerificationRelationships(relationships []Verificati
 		if !relationship.hasDedicatedMethod() {
 			// if the relationship isn't a dedicated verification method,
 			// the referenced verification method must be presented in the 'verificationMethod' property.
-			if _, ok := doc.VerificationMethodByID(relationship.VerificationMethodID); !ok {
+			if _, ok := doc.VerificationMethodByID(relationship.GetVerificationMethodId()); !ok {
```

Comment:
I think `doc.VerificationMethodByID` should  change to `doc.VerificationMethodById`.
Just like the others. :)

Reply:
Yeah, but Go recommends to use a consistent case for initialisms or acronyms: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
Sadly, the protobuf generates them as camel-case stupidly. But, I think it's good for us to follow Go's rules for our source codes.

Reply:
Aha. I got it.